host-bmc/utils.cpp

```diff
@@ -175,8 +175,21 @@ void updateEntityAssociation(
         fs::path path{"/xyz/openbmc_project/inventory"};
         std::deque<std::string> paths{};
         pldm_entity node_entity = pldm_entity_extract(entity);
-        auto node = pldm_entity_association_tree_find(entityTree, &node_entity,
-                                                      false);
+        uint16_t remoteContainerId =
+            pldm_entity_node_get_remote_container_id(entity);
+        /* If the logical bit is set in the host containerId, cosider this as
+         * host entity and find in host node by setting the is_remote to true)
+         */
+        info(
```

Comment:
do we need this trace? won't it fill out the journal?

Reply:
This traces gets printed nearly 30 times during IPL in system with splitter. Debugging will get easier with the trace.  We can keep this trace for few days, once we are sure that everything is working fine we can remove it.